Core/Libraries/Source/EABrowserEngine/BrowserEngine.idl

```diff
@@ -2,6 +2,9 @@
 // 
 // typelib filename: <could not determine filename>
 
+// NOTE: Import of oaidl.idl has been added to fix comilation error with widl:
```

Comment:
compilation

Reply:
I'll fix that. (I should do more checking for typos.)